### DISCUSSION TITLE
Invoice Download option in Deepgram login

### DISCUSSION THREAD

**Discussion by pranavzinzuwadiaatcrimsoni** [deepgram/community#422] [General help] [answered]
Hello, 

We are using the "Pay As You Go" plan and payment is getting deducted from our card based on usage. However we are not getting Invoices and Receipt for these payments.

Please help us to find option / page, from where we can download Invoice and Receipt of the payment made.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @pranavzinzuwadiaatcrimsoni, if you log into your Deepgram console account, and go to Settings -> Billing & Plans, then you can access receipts for your credit card swipes.

We currently don't offer the ability to download invoices on the Pay As You Go plan. If you have a need for them, then if you can provide information to identify your account (such as a company name, email address, project ID, or request ID), then I'll connect you with our finance team to create an invoice to meet your needs.

**Reply by pranavzinzuwadiaatcrimsoni** [reply]
Hi @jkroll-deepgram , 
Thank you for your quick response. 

Request IDs are 3800 & 3777. 
If more information is required, please share your email id.

**Reply by jkroll-deepgram** [reply]
Hi @pranavzinzuwadiaatcrimsoni, you can please reach out to me at -

**Comment by FabriceACohen** [comment]
Hello,
Same thing I need the invoice for the pay as you go payment I made.
I don't want to expose my information to the public. So how can I contact you ?
Thank you.

**Reply by jkroll-deepgram** [reply]
Hi @FabriceACohen, I see the Deepgram project under your name. I'll reach out to the associated email.

**Comment by martin-cycle** [comment]
Hello,

Same request here, I need all the invoices for the pay as we go payements we made. Here are my informations:

Company name: Cycle App
Email address: martin@cycle.app
Project ID: d1febf2b-b8fa-4f37-9f1a-31dc599264e0

Thank you!

**Reply by jkroll-deepgram** [reply]
Hi @martin-cycle, I've sent an email including you and Deepgram's billing department to initiate the invoice request!

**Comment by zakaria-hamane** [comment]
Hello,

Same request here, I need all the invoices for the pay as we go payements we made. Here are my informations:

Company name: Leyton
Email address: [datalab@leyton.com](mailto:datalab@leyton.com)
Project ID: 33cb906a-d14f-4269-b5b8-3353a4c77e9a

Thank you!

**Comment by victorcoliveira** [comment]
Hello,

Same request here, I need all the invoices for the pay as we go payements we made. Here are my informations:

Email address: [victor@meetime.com.br](mailto:victor@meetime.com.br)
Project ID: 4c1f5bef-f8e4-4feb-8b92-a841263797e5

Thank you!

**Comment by Na100Procent** [comment]
Hello
I have the same request as the rest 
Email address: biuro@revele.pl
Project ID: c5b5a68a-42d7-47bf-b26c-80a9810630a8

We need to document every spending in our company. 
Best regards

**Comment by paulbalogh** [comment]
Same request here:

Email: paul@line-21.com
Project id: c440f8f0-9dfa-4837-a3a9-cda97220a3ca

**Comment by ujlm** [comment]
Same request here:

Email: deepgram@ujlm.anonaddy.com
Project id: c74f70fd-baa2-4e51-9775-d74957a8904c

**Comment by Okirido** [comment]
Hello,

Same request here, I need all the invoices for the pay as we go payements we made. Here are my informations:

Email address: plataformaopasuite@gmail.com
Project ID: bee6b8b6-4784-45df-87a9-cf539b2468f6

Thank you!

**Comment by ujlm** [comment]
I'd lika also an invoice for my latest payment:

Email: deepgram@ujlm.anonaddy.com
Project id: c74f70fd-baa2-4e51-9775-d74957a8904c

It would be great if we could just download the invoices from the console though.

**Reply by jkroll-deepgram** [reply]
@ujlm, understood. We have the task of enabling downloading of invoices from the console in our backlog. Unfortunately I don't have an estimated date when that feature will be available. I've followed up via email.

**Comment by ospaarmann** [comment]


**Reply by jkroll-deepgram** [reply]
Hi @ospaarmann, we now have a new invoice form! If you'd like, you may edit/delete your comment to remove your info.

Instead, please fill out the [invoice request form](https://docs.google.com/forms/d/e/1FAIpQLScX7Y95x2oTsW_sKrY85IplJoNfnNo5-DW1sQw7Arzw6CFoLw/viewform?pli=1).

**Comment by jkroll-deepgram** [comment]
For future invoice requests, rather than posting in this thread, please submit this [invoice request form](https://docs.google.com/forms/d/e/1FAIpQLScX7Y95x2oTsW_sKrY85IplJoNfnNo5-DW1sQw7Arzw6CFoLw/viewform?pli=1).
